src/modules/edc-demo/pages/contract-viewer/contract-viewer.component.ts

```diff
@@ -140,7 +143,7 @@ export class ContractViewerComponent implements OnInit {
    * @param assetId Asset ID of the asset that is associated with the contract.
    */
   private getOfferedAssetForId(assetId: string): Observable<Asset> {
-    return this.catalogService.getContractOffers()
+    return this.catalogService.getContractOffers(this.url, this.ownAssets)
```

Comment:
`this.ownAssets` wird nirgendwo assigned, kann also direkt durch `false` ersetzt werden.